frontend/package.json

```diff
@@ -37,6 +37,7 @@
 		"next-themes": "^0.3.0",
 		"react": "^18.3.1",
 		"react-dom": "^18.3.1",
+		"sonner": "^1.5.0",
```

Comment:
Ui, fancy :D Ich hab einfach den standard shadcn toast benutzt, mit ein bissl config ist der auch in Ordnung.
Sonner kannte ich garnicht

Reply:
Ist tatsächlich der "neue" Toast von shadcn
https://ui.shadcn.com/docs/components/sonner